### DISCUSSION TITLE
Results of ERAD course survey

### DISCUSSION THREAD

**Discussion by jfigui** [openradar/erad2022#80] [General]
Dear all,
[Here](https://www.surveymonkey.com/stories/SM-vf3AgOdRhvIhoxoXv_2BByqA_3D_3D/) you can find the results of the survey.

**Additional comments:**
**Q1:** perhaps somewhere in between academy and industry as I work for a government lab
**Q2:** As a public agency, it has been required to abandon software under license
**Q4:** The structure of the course was well planned. Very interesting idea of looking at individual sofware and, then, how you can combine them. The time was too short to understand all the software presented. Beyond the connection problems, which did not help, I found it difficult to follow the explanations and, simultaneously try out the notebooks.
**Q5:** The presentations and notebooks were well prepared, but, as mentioned above, there was not enough time to be able to independently try out what we had just heard.
**Q8:** My goal was to understand, in addition to the software I already use, what options are available to those who have to work with radar data, both from an operational and research perspective.
**Q9:** Unfortunately, it is not so easy for me to contribute.
**Q10:**  
really enjoyed the course and the material presented which gave me a very nice overview of the topic and a nice material resource. The examples (notebooks) presented were also very valuable and interesting. Having maybe a bit more time to go through the different examples that were presented would have been nice to better understand the specificities of each library and their differences. Thanks again for the nice course!
I would recommend it to future students, academics, etc.
The course really touched on so many aspects and so many possible software available. The amount of information was really great and not enough time to fully digest and understand everything.

**Comment by jfigui** [comment]
My take on the survey:

- We had 12 answers from 20 participants so we can say it is representative.
- Most participants have used either wradlib or Py-ART in the past. Very few have used the other packages
- Participants are quite satisfied with the contents and structure of the course
- The amount of time for the course was considered correct but longer format may be welcomed
- At least some of the participants plan to adopt new software

**Reply by scollis** [reply]
Agreed. This is nice feedback.

**Reply by kmuehlbauer** [reply]
@jfigui Thanks! That's great news. I'd announce this at the discourse, https://openradar.discourse.group/, if that's OK.

**Reply by jfigui** [reply]
@kmuehlbauer , please go ahead.

**Reply by kmuehlbauer** [reply]
https://openradar.discourse.group/t/ann-erad2022-open-radar-short-course-survey-results/74

**Reply by DanielMichelson** [reply]
Happy to see the effort made to gather this feedback, and that it is positive! Well done everyone!

**Comment by jfigui** [comment]
Former and future participants, please feel free to add your comments to this thread.
